### ISSUE TITLE
sh argument doees not work properly in automorphisms() and canonical_permutation()

### ISSUE THREAD

**Issue by szhorvat** [igraph/rigraph#518]
**Describe the bug**

`sh` argument does not work properly in `automorphisms()` and `canonical_permutation()`. It only accepts the default value, which is `'fm'`.

**To reproduce**

```
> canonical_permutation(make_ring(5),sh='fs')
Error in match.arg(arg = arg, choices = choices, several.ok = several.ok) : 
  'arg' should be one of “fm”
```

```
> automorphism_group(make_ring(5),sh='fs')
Error in match.arg(arg = arg, choices = choices, several.ok = several.ok) : 
  'arg' should be one of “fm”
```

```
> automorphisms(make_ring(5),sh='fs')
Error in match.arg(arg = arg, choices = choices, several.ok = several.ok) : 
  'arg' should be one of “fm”
```

**Notes**

This should be checked for all Bliss functions. Is the following expected?

```
> isomorphic(make_ring(10), make_star(10, mode='undirected'), method='bliss', sh1='fs', sh2='fs')
Error in graph.isomorphic.bliss(graph1, graph2, ...) : 
  unused arguments (sh1 = "fs", sh2 = "fs")
```

The docs seem to say that the optional args `sh1` and `sh2` are available for Bliss.

Finally, the docs for `automorphisms` and related functions are like this:

> automorphisms(graph, colors, sh = "fm")

Should this not be the following instead?

> automorphisms(graph, colors=NULL, sh = "fm")

I'm not clear about how this is normally done in R, @ntamas .

**Version information**

1.3.0

**Comment by ntamas** [comment]
It seems like the BLISS implementation in igraph 0.9.7 (i.e. the C core) allows only a single `sh` argument for BLISS, not `sh1` and `sh2`. I'm fixing this as well as the handling of the default argument.

**Comment by szhorvat** [comment]
Indeed, allowing separate splitting heuristics was a bug because the canonicalization depends on the splitting heuristic.
